consensus/vbft/msg_types.go

```diff
@@ -79,6 +79,16 @@ func (msg *blockProposalMsg) Verify(pub keypair.PublicKey) error {
 	if !signature.Verify(pub, hash[:], sig) {
 		return fmt.Errorf("failed to verify block sig")
 	}
+	if msg.Block.CrossChainMsg != nil {
+		cSig, err := signature.Deserialize(msg.Block.CrossChainMsg.SigData[0])
```

Comment:
check if msg.Block.CrossChainMsg.SigData is nil?

Reply:
why need check?